login-workflow/src/contexts/RegistrationContext/types.ts

```diff
@@ -25,10 +67,39 @@ export type RegistrationUIActions = {
 };
 
 export type RegistrationContextProviderProps = {
+    /**
+     * An object of functions that are used to manage the authentication workflow
+     * @object RegistrationUIActions
```

Comment:
This type is changing, but as written it should be
```suggestion
     * @returns RegistrationUIActions
```